PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/azure-intellij-plugin-azure-sdk.xml

```diff
@@ -23,5 +23,19 @@
                 implementationClass="com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool.MavenProjectInspection">
             <class>com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool.MavenProjectInspection</class>
         </localInspection>
+
+        <localInspection
+                language="JAVA"
+                shortName="ServiceBusReceiverAsyncClientCheck"
+                displayName="ServiceBusReceiverAsyncClient Check"
```

Comment:
Does the DisplayName show up in the editor? If so, maybe "Low level ServiceBusReceiverAsyncClient check"?, the phrase _Low level_ can help catch attention, indicating client is not for common use cases.